objectRecognitionSource/src/test/java/io/vantiq/extsrc/objectRecognition/neuralNet/TestYoloProcessor.java

```diff
@@ -1735,32 +1734,34 @@ public void testSuppressNullValuesHelper(boolean suppressNullValues) throws Inte
 
         // Wait for 15 seconds while the source polls for frames from the data source and stores data in type
         Thread.sleep(15000);
-
-        // Make sure that appropriate number of entries are stored in type (this means discard policy works, and core is still alive)
-        VantiqResponse response = vantiq.select(testTypeName, null, null, null);
-        ArrayList responseBody = (ArrayList) response.getBody();
-
-        // If suppressNullValues is set to true, we shouldn't have any results stored in our type
-        if (suppressNullValues) {
-            assertEquals("Get responseBody content when none expected: " + responseBody, 
-                    0, responseBody.size());
-        } else {
-            // Otherwise, we should have some results and they should be empty arrays
-            assert responseBody.size() > 0;
-
-            for (int i = 0; i < responseBody.size(); i++) {
-                JsonObject resultObject = (JsonObject) responseBody.get(i);
-                assert resultObject.get("results") instanceof JsonArray;
-                assertEquals("Get resultObject.results content when none expected: " + responseBody,
-                        0, ((JsonArray) resultObject.get("results")).size());
+        try {
+            // Make sure that appropriate number of entries are stored in type (this means discard policy works, and core is still alive)
+            VantiqResponse response = vantiq.select(testTypeName, null, null, null);
+            ArrayList responseBody = (ArrayList) response.getBody();
+
+            // If suppressNullValues is set to true, we shouldn't have any results stored in our type
+            if (suppressNullValues) {
+                assertEquals("Get responseBody content when none expected: " + responseBody,
+                        0, responseBody.size());
+            } else {
+                System.out.println("Found response: " + responseBody);
+                // Otherwise, we should have some results and they should be empty arrays
+                assert responseBody.size() > 0;
+
+                for (int i = 0; i < responseBody.size(); i++) {
+                    JsonObject resultObject = (JsonObject) responseBody.get(i);
+                    assert resultObject.get("results") instanceof JsonArray;
+                    assertEquals("Get resultObject.results content when none expected: " + responseBody,
+                            0, ((JsonArray) resultObject.get("results")).size());
+                }
             }
+        } finally {
```

Comment:
added try so this test cleans up after failure.  It wasn't, causing its counterpart to be skipped.